prowler/compliance/aws/aws_account_security_onboarding_aws.json

```diff
@@ -0,0 +1,1155 @@
+        "route53_domains_transferlock_enabled"
+      ]
+    },
+    {
+      "Id": "Apply custom threat list for GuardDuty to alert on access to DoH servers",
+      "Description": "Implement a customized threat list within GuardDuty to generate alerts when there is access to Domain Name System over HTTPS (DoH) servers.",
+      "Attributes": [
+        {
+          "Section": "R53 DNS Resolver Firewall",
+          "Service": "Route53"
+        }
+      ],
+      "Checks": [
+        "route53_domains_privacy_protection_enabled",
+        "route53_domains_transferlock_enabled"
+      ]
+    },
+    {
+      "Id": "Enable Shield Advanced subscription for public facing account",
+      "Description": "Activate the Shield Advanced subscription for the publicly accessible account.",
+      "Attributes": [
+        {
+          "Section": "Shield Advanced",
+          "Service": "Shield"
+        }
+      ],
+      "Checks": [
+        "shield_advanced_protection_in_route53_hosted_zones",
+        "shield_advanced_protection_in_classic_load_balancers",
+        "shield_advanced_protection_in_internet_facing_load_balancers",
+        "shield_advanced_protection_in_cloudfront_distributions"
+      ]
+    },
+    {
+      "Id": "Export metrics in centralized collector",
+      "Description": "Exporting metrics to a centralized collector for data aggregation and analysis.",
+      "Attributes": [
+        {
+          "Section": "Shield Advanced",
+          "Service": "CloudWatch"
+        }
+      ],
+      "Checks": [
+        "shield_advanced_protection_in_route53_hosted_zones",
+        "shield_advanced_protection_in_classic_load_balancers",
+        "shield_advanced_protection_in_internet_facing_load_balancers",
+        "shield_advanced_protection_in_cloudfront_distributions"
+      ]
+    },
+    {
+      "Id": "Create DDoS battle card with main info about protected services",
+      "Description": "Prepare a Detailed Distributed Denial of Service (DDoS) Battle Card Encompassing Key Information Regarding Safeguarded Services.",
+      "Attributes": [
+        {
+          "Section": "Shield Advanced",
+          "Service": "Shield"
+        }
+      ],
+      "Checks": [
+        "shield_advanced_protection_in_route53_hosted_zones",
+        "shield_advanced_protection_in_classic_load_balancers",
+        "shield_advanced_protection_in_internet_facing_load_balancers",
+        "shield_advanced_protection_in_cloudfront_distributions"
+      ]
+    },
+    {
+      "Id": "CFD + ALB + secret rotation architecture",
+      "Description": "Designing an Architecture for Computational Fluid Dynamics (CFD), Application Load Balancing (ALB), and Secret Rotation Integration",
+      "Attributes": [
+        {
+          "Section": "WAFv2",
+          "SubSection": "Deploy WAF setup for each public web service",
+          "Service": "WAF",
+          "Type": "Must"
+        }
+      ],
+      "Checks": []
```

Comment:
Please add the following the following checks for WAF:
> [apigateway_restapi_waf_acl_attached] Check if API Gateway Stage has a WAF ACL attached. - apigateway [medium]
[cloudfront_distributions_using_waf] Check if CloudFront distributions are using WAF. - cloudfront [medium]
[elbv2_waf_acl_attached] Check if Application Load Balancer has a WAF ACL attached. - elbv2 [medium]

Reply:
Done!